swagger.yaml

```diff
@@ -0,0 +1,120 @@
+swagger: "2.0"
+info:
+  description: "Task Management System"
+  version: "0.0.1"
+  title: "Task Scheduler"
+host: "localhost:8080"
+basePath: "/"
+schemes:
+  - "http"
+consumes:
+  - "application/json"
+produces:
+  - "application/json"
+paths:
+  /task:
+    post:
+      operationId: "addTask"
+      parameters:
+        - in: "body"
+          name: "task"
+          description: "task's details"
+          required: true
+          schema:
+            $ref: "#/definitions/Task"
+      responses:
+        201:
+
+          description: "task added"
+          schema:
+            $ref: "#/definitions/Task"
+        409:
+          description: "task already exists"
+        500:
+          description: "internal server error"
+    put:
+      operationId: "updateTask"
+      parameters:
+        - in: "body"
+          name: "task"
+          required: true
+          description: "task details"
+          schema:
+            $ref: "#/definitions/Task"
+      responses:
+        200:
+          description: "task updated"
+          schema:
+            $ref: "#/definitions/Task"
+        500:
```

Comment:
do you think we need 404 response in update task request?

Reply:
yes will add it